api/test/test_sample.py

```diff
@@ -54,3 +70,51 @@ def test_get_sample(self):
 
         self.assertEqual(get_response.status_code, 200)
         self.assertEqual(list(get_response.data.keys()), self.sample_keys)
+
+    def test_single_gene_query(self):
+        client = APIClient()
+
+        get_response = client.get('/samples?disease=' + self.disease1.acronym +
+                                  '&mutations__gene=' +
+                                  str(self.gene2.entrez_gene_id))
```

Comment:
This isn't strictly necessary, but it will make reading and maintaining this easier if you change this to something like:
```
get_response = client.get('/samples',
  {'disease': self.disease1.acronym,
   'mutations__gene': str(self.gene2.entrez_gene_id)})
```

Reply:
If you do decide to change it to something like this, it can also be done for the subsequent tests.

Reply:
fancy! seemed like that had to be possible 👍